$NetBSD: patch-CMakeLists.txt,v 1.1 2023/06/05 20:35:45 abs Exp $

Ensure we use ${CMAKE_INSTALL_PREFIX}/share rather than /usr/share,
as otherwise the "Welcome to LXQt" dialog cannot find any window
managers to suggest

--- CMakeLists.txt.orig	2023-04-15 15:50:30.000000000 +0000
+++ CMakeLists.txt
@@ -71,8 +71,8 @@ set(PREDEF_XDG_DATA_DIRS "$XDG_DATA_HOME
 if(NOT("${LXQT_DATA_DIR}" MATCHES "^/usr(/local)?/share$"))
     set(PREDEF_XDG_DATA_DIRS "${PREDEF_XDG_DATA_DIRS}:${LXQT_DATA_DIR}")
 endif()
-set(PREDEF_XDG_DATA_DIRS "${PREDEF_XDG_DATA_DIRS}:/usr/local/share:/usr/share")
-set(PREDEF_XDG_CONFIG_DIRS "/etc:${LXQT_ETC_XDG_DIR}:/usr/share")
+set(PREDEF_XDG_DATA_DIRS "${PREDEF_XDG_DATA_DIRS}:/usr/local/share:${CMAKE_INSTALL_PREFIX}/share")
+set(PREDEF_XDG_CONFIG_DIRS "/etc:${LXQT_ETC_XDG_DIR}:${CMAKE_INSTALL_PREFIX}/share")
 configure_file(startlxqt.in startlxqt @ONLY)
 install(PROGRAMS
     "${CMAKE_CURRENT_BINARY_DIR}/startlxqt"
